app/build.gradle

```diff
@@ -15,7 +15,7 @@ android {
         targetSdkVersion 30
 
         versionCode 3
-        versionName "0.3"
+        versionName "0.0.3"
```

Comment:
I did this since we need a 3 component version for Vulkan, I did make it `0.0.3` rather than `0.3.0` by mistake but does this matter? It's sort of abstract anyway but I could change it to that if you really want it to be `0.3.0`

Reply:
personally prefer 1.0.0 since that's pretty neat